build.sbt

```diff
@@ -1,27 +1,85 @@
-enablePlugins(ScalaJSPlugin)
-
-enablePlugins(ScalaJSBundlerPlugin)
 
 // @TODO[Security] Is this a good idea to leave this here long term?
 // resolvers += Resolver.sonatypeRepo("snapshots")
 
-libraryDependencies ++= Seq(
-  "com.raquo" %%% "airstream" % "0.10.1",
-  "com.raquo" %%% "domtypes" % "0.10.1",
-  "com.raquo" %%% "domtestutils" % "0.12.0" % Test,
-  "org.scalatest" %%% "scalatest" % "3.1.1" % Test
-)
 
-scalacOptions ++= Seq("-deprecation", "-feature", "-language:higherKinds", "-language:implicitConversions")
+val Scala212 = "2.12.10"
+val Scala213 = "2.13.1"
+val scalaVersions = Seq(Scala212, Scala213)
+
+lazy val jsdocs = project
+  .settings(
+   libraryDependencies += "org.scala-js" %%% "scalajs-dom" % "1.1.0",
+   scalaVersion := Scala212,
```

Comment:
Is anything preventing us from using 2.13?

Reply:
Hm, I think it's an oversight in mdoc - js modifier is only published for 2.12. I'm going to [rectify that](https://github.com/scalameta/mdoc/pull/385)

Until that's published, I can force 2.12 during the site building, but keep laminar's own default Scala version at 2.13 (not that it matters too much, given it's cross-publishing)